src/styles/themes/ThemeColors.ts

```diff
@@ -0,0 +1,85 @@
+type HexColor = `#${string}`;
+type Color = HexColor | `rgba(${number}, ${number}, ${number})` | `rgba(${number}, ${number}, ${number}, ${number})` | 'transparent';
```

Comment:
```suggestion
type Color = HexColor | `rgb(${number}, ${number}, ${number})` | `rgba(${number}, ${number}, ${number}, ${number})` | 'transparent';
```

Reply:
Also, by defining the types in this way we are going to restrict all the other possible color representations -> https://reactnative.dev/docs/colors#color-representations

Is it intended?

Reply:
I'm thinking if we even need such strict type for colors? Usually `string` was sufficient enough. Wdyt @fabioh8010?

Reply:
I agree @blazejkustra , it's not common to type colors so strictly. Both RN and third-party components usually just type it as `string`.

Reply:
You guys are right! 👍 I just replaced this type with `string`.  Makes things a lot easier